app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container.rb

```diff
@@ -22,6 +24,33 @@ class ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer
     end
   end
 
+  def self.params_for_create(_ems)
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'name',
+          :id         => 'name',
+          :label      => _('Container Name'),
+          :isRequired => true,
+          :validate   => [{:type => 'required'}],
+        },
+        {
+          :component    => 'select',
+          :name         => 'provider_region',
+          :id           => 'provider_region',
+          :label        => _('Region'),
+          :isRequired   => true,
+          :includeEmpty => true,
+          :validate     => [{:type => 'required'}],
+          :options      => ManageIQ::Providers::Amazon::Regions.all.map do |region|
+            {:label => region[:description], :value => region[:name]}
+          end
```

Comment:
NOTE we can do this in a follow-up but this is almost exactly what we do for the region selection when creating a provider so we should be able to deduplicate this